File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.py
class Solution(object):
    def removeDuplicates(self , nums):
        l=r=1
        size = len(nums)
        
        while r < size :
            if nums[r] != nums[r-1]:
                nums[l] = nums[r]
                l += 1
            r += 1
            
        return l 

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.c
int removeDuplicates(int* nums, int numsSize) {
	int p = 1, r = 1;
	for (r; r < numsSize; r++){
		if (nums[p] > nums[p - 1]) p++;
		else {
			if (nums[r] > nums[r - 1]) {
				nums[p] = nums[r];
				p++;
			}
		}

	}
    return p;
}
